includes/classes/Screen.php

```diff
@@ -56,7 +56,7 @@ public function determine_screen() {
 		// VIP: We removed the block about returning false depending on network admin or in network mode && not network admin
 
 		// phpcs:disable WordPress.Security.NonceVerification
-		if ( ! empty( $_GET['page'] ) && false !== strpos( $_GET['page'], 'elasticpress' ) ) {
+		if ( ! empty( $_GET['page'] ) && false !== strpos( sanitize_key( $_GET['page'] ), 'elasticpress' ) ) {
```

Comment:
While this works, the call to `sanitize_key()` incurs some performance penalty. If we only check for a substring, I would not call `sanitize_key()`.

Reply:
@sjinks This as well was pulled from upstream